test/cmd/request-timeout.sh

```diff
@@ -39,7 +39,9 @@ run_kubectl_request_timeout_tests() {
 
   ## check --request-timeout on 'get pod' with --watch
   output_message=$(kubectl get pod valid-pod --request-timeout=1 --watch --v=5 2>&1)
-  kube::test::if_has_string "${output_message}" 'Timeout'
+  if ! kube::test::if_has_string "${output_message}" 'Timeout'; then
```

Comment:
From the troubleshooting results, it seems that we need to find out the keywords of context deadline exceeded in addition to Timeout to ensure that this occasional error does not occur.